backend/app.ts

```diff
@@ -40,6 +40,14 @@ import routes from "./routes/index.js";
 
 app.use(routes);
 
+declare global {
+	export interface Error {
+		title?: string;
+		status?: number;
+		errors?: { [key: string]: any };
+	}
+}
+
```

Comment:
Nice job exporting Errors. Simplifies error handling/messages throughout the project. (Very cleann )